[2022-08-29 16:20:04,831] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.process_user scheduled__2022-08-28T00:00:00+00:00 [queued]>
[2022-08-29 16:20:04,843] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.process_user scheduled__2022-08-28T00:00:00+00:00 [queued]>
[2022-08-29 16:20:04,843] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 16:20:04,843] {taskinstance.py:1357} INFO - Starting attempt 5 of 5
[2022-08-29 16:20:04,844] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 16:20:04,863] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): process_user> on 2022-08-28 00:00:00+00:00
[2022-08-29 16:20:04,871] {standard_task_runner.py:52} INFO - Started process 3255 to run task
[2022-08-29 16:20:04,877] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'process_user', 'scheduled__2022-08-28T00:00:00+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpe6j_t1wt', '--error-file', '/tmp/tmpp02tuxm8']
[2022-08-29 16:20:04,879] {standard_task_runner.py:80} INFO - Job 101: Subtask process_user
[2022-08-29 16:20:05,041] {task_command.py:369} INFO - Running <TaskInstance: user_processing.process_user scheduled__2022-08-28T00:00:00+00:00 [running]> on host 7f0ff65cfbf7
[2022-08-29 16:20:05,191] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=process_user
AIRFLOW_CTX_EXECUTION_DATE=2022-08-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=5
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-28T00:00:00+00:00
[2022-08-29 16:20:05,223] {python.py:173} INFO - Done. Returned value was: None
[2022-08-29 16:20:05,246] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=process_user, execution_date=20220828T000000, start_date=20220829T162004, end_date=20220829T162005
[2022-08-29 16:20:05,303] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-29 16:20:05,383] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
